examples/metrics/kafka-metrics.yaml

```diff
@@ -65,6 +65,8 @@ spec:
   kafkaExporter:
     topicRegex: ".*"
     groupRegex: ".*"
+    topicExcludeRegex: "^$"
+    groupExcludeRegex: "^$"
```

Comment:
You should not change anything in `examples/`, `install/` or `helm-charts/`. All changes should be done to `packaging/` only.